.circleci/config.yml

```diff
@@ -1,21 +1,27 @@
 version: 2.1
-
 orbs:
-  architect: giantswarm/architect@0.8.4
+  architect: giantswarm/architect@0.18.1
 
 workflows:
-  test:
+  version: 2
+  build:
     jobs:
-      - architect/go-test:
-          name: go-test
+      - architect/push-to-app-catalog:
+          name: e2esetup-certs-to-app-catalog
+          app_catalog: "default-test-catalog"
```

Comment:
There is no need to put this chart into official `default` appcatalog. so I assigned it as well to `default-test`.

Reply:
Good plan. 👍